libsrc/Makefile

```diff
@@ -91,13 +96,14 @@ define INSTALL_recipe
 
 $(if $(prefix),,$(error variable `prefix' must be set))
 $(INSTALL) -d $(DESTDIR)$(datadir)/$(dir)
-$(INSTALL) -m644 ../$(dir)/*.* $(DESTDIR)$(datadir)/$(dir)
+$(INSTALL) -m0644 ../$(dir)/*.* $(DESTDIR)$(datadir)/$(dir)
```

Comment:
I must admit that I'm no friend of those "while I'm at it I'll just do this minor thing as well" changes. And here we have already a reason why I think so. You didn't do the same change to the doc/Makefile. Please add it there too or remove it here.